README.md

```diff
@@ -2,46 +2,51 @@
 
 The documentation content home for https://docs.quantum.ibm.com (excluding API reference).
 
-# Improving IBM Quantum & Qiskit Documentation
+Refer to:
 
-Maintaining up-to-date documentation is a huge challenge for any software project, especially in a field like quantum computing where the pace at which advances in new research and technological capabilities happen incredibly fast. As a result, we greatly appreciate any who take the time to support us in keeping this content accurate and up to the highest quality standard possible to benefit the broadest range of users.
+- Our [MDX guide](./mdx-guide.md) for how to write documentation and use our variant of markdown.
+- Our [style guide](./style-guide.md) for technical writing guidance.
+
+## Improving IBM Quantum and Qiskit documentation
+
+Maintaining up-to-date documentation is a huge challenge for any software project, but especially for a field like quantum computing, because advances in new research and technological capabilities come at a face pace. As a result, we greatly appreciate anyone who takes the time to support us in keeping this content accurate and up to the highest quality standard possible, to benefit the broadest range of users.
```

Comment:
```suggestion
Maintaining up-to-date documentation is a huge challenge for any software project, but especially for a field like quantum computing, because advances in new research and technological capabilities come at a fast pace. As a result, we greatly appreciate anyone who takes the time to support us in keeping this content accurate and up to the highest quality standard possible, to benefit the broadest range of users.
```